python-proto-example.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
```

Comment:
this file should be removed entirely, and also the .proto files should be removed from the specification folder